### ISSUE TITLE
Infinite food and drink from the still and candy machine

### ISSUE THREAD

**Issue by darkpaladin109** [UristMcStation/UristMcStation#486]
As the title says. It seems if you take something, like say, a potato, and use it on either of those two machines, and put it into some sort of bag before selecting what item to make, it creates the item without using up what you used to make it.

**Comment by scrdest** [comment]
The kitchen machines in general are... not the paragons of good code.

**Comment by Mapleguy555** [comment]
All the goofball kitchen machines can be induced to use "ghost" ingredients by clicking on them multiple times.

**Comment by scrdest** [comment]
Yes, I know. See my comment above. BS12 has ported at least some of the kitchen machines from a different server that implemented goofcode with a number of fixes (although some less !!FUN!!), so they can be replaced post-Summerge

**Comment by scrdest** [comment]
'Fixed' for candy machine by way of replacing it with the saner Bay variant. Still still problematic.
